tests/testthat/test-shinytest2-tm_g_association.R

```diff
@@ -0,0 +1,101 @@
+app_driver_tm_g_association <- function() {
+  data <- within(teal.data::teal_data(), {
+    require(nestcolor)
+    require(ggplot2)
+    CO2 <- CO2 # nolint: object_name.
+    factors <- names(Filter(isTRUE, vapply(CO2, is.factor, logical(1L))))
+    CO2[factors] <- lapply(CO2[factors], as.character) # nolint: object_name.
+  })
+  teal.data::datanames(data) <- c("CO2")
+
+  init_teal_app_driver(
+    data = data,
+    modules = tm_g_association(
```

Comment:
@m7pr, you suggested adding all the arguments even if there is a default value, so that if an argument is removed, we can catch it. Let's add all the arguments.
![image](https://github.com/insightsengineering/teal.modules.general/assets/6700955/e7c9e934-4c97-4af1-8016-9483b95e6ab1)